core/src/smartcontracts/wasm.rs

```diff
@@ -67,8 +74,12 @@ mod import {
     pub const QUERY_AUTHORITY_FN_NAME: &str = "query_authority";
     /// Name of the imported function to query event that triggered the smart contract execution
     pub const QUERY_TRIGGERING_EVENT_FN_NAME: &str = "query_triggering_event";
-    /// Name of the imported function to query operation that is to be verified
-    pub const QUERY_OPERATION_TO_VALIDATE_FN_NAME: &str = "query_operation_to_validate";
+    /// Name of the imported function to query transaction that is to be verified
+    pub const QUERY_TRANSACTION_TO_VALIDATE_FN_NAME: &str = "query_transaction_to_validate";
+    /// Name of the imported function to query instruction that is to be verified
+    pub const QUERY_INSTRUCTION_TO_VALIDATE_FN_NAME: &str = "query_instruction_to_validate";
+    /// Name of the imported function to request query that is to be verified
+    pub const QUERY_QUERY_TO_VALIDATE_FN_NAME: &str = "query_query_to_validate";
```

Comment:
Ah, here they are. Would be great to restructure these consts so that it is possible to reuse them even in the validator macros.

Reply:
Created #3757